governance-app/src/signCreatePlt.c

```diff
@@ -0,0 +1,149 @@
+#include <os.h>
+
+#include "base58check.h"
+#include "responseCodes.h"
+#include "sign.h"
+#include "util.h"
+
+static signCreatePltContext_t *ctx = &global.signCreatePltContext;
+static tx_state_t *tx_state = &global_tx_state;
+
+UX_STEP_NOCB(
+    ux_sign_create_plt_token_symbol,
+    bnnn_paging,
+    {"Token Symbol", (char *) global.signCreatePltContext.tokenId});
```

Comment:
```suggestion
    {"Token ID", (char *) global.signCreatePltContext.tokenId});
```

This is fixed in #107